assets/js/components.js

```diff
@@ -62,17 +62,21 @@ $(function() {
     accordion($(this));
   });
 
-
   var footerAccordion = function() {
     if (window.innerWidth < 600) {
-        $('.usa-footer-big nav ul').addClass('hidden');
 
-        $('.usa-footer-big nav').on('click', 'h3', function() {
-          $(this).parent().removeClass('hidden')
-          .siblings().addClass('hidden');
-        });
+      $('.usa-footer-big nav ul').addClass('hidden');
+
+      $('.usa-footer-big nav h3').bind('click', function() {
```

Comment:
I saw this yesterday. Technically, this event should only be bound if it hasn't been bound yet, or you'll bind two events. I guess we could just unbind and rebind:

`$('.usa-footer-big nav h3').unbind('click')`